jenkins-plugin/src/test/java/jenkins/scm/impl/mock/SampleDVCSRepoRuleUtilsTest.java

```diff
@@ -0,0 +1,46 @@
+package jenkins.scm.impl.mock;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.File;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * @author <a href="mailto:cleclerc@cloudbees.com">Cyrille Le Clerc</a>
+ */
+public class SampleDVCSRepoRuleUtilsTest {
+
+    @Test
+    public void testCopy() throws Throwable {
+        System.out.println("SampleDVCSRepoRuleUtilsTest new File('.'): " + new File(".").getAbsolutePath());
+
+        AbstractSampleDVCSRepoRule sampleDVCSRepoRule = new AbstractSampleDVCSRepoRule() {
+            @Override
+            public void init() throws Exception {
+
+            }
+        };
+
+        sampleDVCSRepoRule.before();
+
+         File mavenProjectRoot = new File("src/test/test-maven-projects/maven-jar-project");
```

Comment:
I initially did this but the maven-surefire-plugin started to complain trying to parse/compile my ".java" files that were located under src/test/resources.

Unfortunately, I did not collect the error message and I can't reproduce.

I'll switch back to src/test/resources and collect details if a problem occur.